tests/mir-opt/inline/inline_retag.rs

```diff
@@ -7,8 +6,19 @@ fn main() {
     println!("{}", bar());
 }
 
-// EMIT_MIR inline_retag.bar.Inline.after.mir
+// EMIT_MIR inline_retag.bar.Inline.diff
```

Comment:
The pre-inlining MIR depends on unwinding config. Using `after.mir` is fine to avoid this.
```suggestion
// EMIT_MIR inline_retag.bar.Inline.after.mir
```